src/templates/_stripView.hbs

```diff
@@ -1,4 +1,7 @@
 <div id="sidebar" class="stripView js-stripView mix-stripView_isHidden">
     <div class="stripView-trigger js-stripView-trigger"></div>
-    <ul id="strips" class='stripView-list nice-scrollbar js-stripView-list'></ul>
+    <div class="stripTitle">Arrivals</div>
+    <ul id="strips-arrivals" class='stripView-list nice-scrollbar js-stripViewArrivals-list'></ul>
```

Comment:
are you using the `id`s at all?  if not, remove them.  if you are, define them as classes instead because we are slowly moving away from the `id`s.

Reply:
No, not using these ids. Removed them from latest commit.